### PR TITLE
Grammar and word smithing: maps and gross-store

### PR DESCRIPTION
**Pull request by IsaacG**

### CODE REVIEW

File: concepts/maps/about.md [hunk 1]
```diff
@@ -46,24 +46,26 @@ or
 >
 > Therefore it's important to initialize a map before using it
 
-Here are some operations that you can use with map
+Here are some operations that you can do with a `map`
```
Comment (junedev) [inline concepts/maps/about.md:49]: Could you please remove the backticks around map everywhere?
The text does not refer to the keyword `map` (where we would use the backticks) but to the English word/general concept "map". That they are both the same word is just coincidence. You wouldn't write "you can do that with a `slice` for example".

**Review by junedev — COMMENTED** [review]
Thanks a lot for already checking all the relevant files here. Just one small comment.

File: concepts/maps/about.md (hunk 1 above)
Comment (IsaacG) [reply concepts/maps/about.md]: Sorry about that. Fixed!

**Review by junedev — APPROVED** [review]

File: concepts/maps/about.md (hunk 1 above)
Comment (junedev) [reply concepts/maps/about.md]: Thanks for the quick fix!

Comment (IsaacG) [reply concepts/maps/about.md]: Thanks for the quick approvals :D

**Pull request merged**